ansible_deployer/command_line.py

```diff
@@ -71,8 +71,8 @@ def parse_options(argv):
                         help='Print debug output.')
     parser.add_argument("--syslog", "-v", default=False, action="store_true", help='Log warnings '
                         'and errors to syslog. --debug doesn\'t affect this option!')
-    parser.add_argument("--limit", "-l", nargs=1, default=[None], metavar="[LIMIT]",
-                        help='Limit task execution to specified host.')
+    parser.add_argument("--limit", "-l", nargs='+', default=[], metavar="LIMIT_1 LIMIT_2",
```

Comment:
If we're going to add this commit we need to add test of functionality too. Is that realy required - won't just running host1,host2 work? If we don't need that I vote to drop it.